internal/cmd/testdata/scripts/issue3603.txtar

```diff
@@ -0,0 +1,2 @@
+exec chezmoi execute-template '{{ "a b" | splitList " " | quoteList }}'
```

Comment:
Please add a comment explaining what is being tested. testscript uses these comments when generating its output.

Also, this test can be moved to `templatefuncs.txtar`.